APMrover2/Rover.h

```diff
@@ -80,6 +80,8 @@
 #include <AC_Fence/AC_Fence.h>
 #include <AP_Proximity/AP_Proximity.h>
 #include <AC_Avoidance/AC_Avoid.h>
+#include <AC_AttitudeControl/AC_PosControl.h>      // Position control library
```

Comment:
I don't think we should need the include or declaration of AC_PosControl.  This library is really meant for multicopter/tradheli position control so shouldn't be required as far as I know.